packages/block-library/src/navigation/use-navigation-menu.js

```diff
@@ -64,6 +65,20 @@ export default function useNavigationMenu( ref ) {
 				),
 				navigationMenu,
 				navigationMenus,
+				canUserUpdateNavigationEntity: ref
+					? canUser( 'update', 'navigation', ref )
+					: undefined,
+				hasResolvedCanUserUpdateNavigationEntity: hasFinishedResolution(
+					'canUser',
+					[ 'update', 'navigation', ref ]
+				),
+				canUserDeleteNavigationEntity: ref
+					? canUser( 'delete', 'navigation', ref )
+					: undefined,
+				hasResolvedCanUserDeleteNavigationEntity: hasFinishedResolution(
+					'canUser',
+					[ 'delete', 'navigation', ref ]
+				),
```

Comment:
Same as above

```suggestion
				canUserDeleteNavigationEntity: ref
					? canUser( 'delete', 'navigation', ref )
					&& hasFinishedResolution(
						'canUser',
						[ 'delete', 'navigation', ref ]
					),
					: undefined,
```

Linter will likely complain about this code formatting, it might need an extra pass

Reply:
I'm confused. You've suggested and then marked as resolved 😕